learning-basics/configure-yaml/src/main/resources/hazelcast-client.yaml

```diff
@@ -0,0 +1,6 @@
+hazelcast-client:
+  import:
+    - hazelcast-client-network-config.yaml
+  group:
+    name: dev-configured-from-yaml
```

Comment:
Do you use it somehow in the code sample?

Reply:
It is needed to be the same in member and client config. It's better to be unique in the samples so no members and clients join from or to instances started by other samples by mistake.